lib/code_corps/github.ex

```diff
@@ -2,10 +2,37 @@ defmodule CodeCorps.Github do
 
   alias CodeCorps.{User, Repo}
 
+  @token_url ""
+  @client_id ""
+  @client_secret ""
+
   @doc """
   Temporary function until the actual behavior is implemented.
   """
-  def connect(user, _code), do: {:ok, user}
+  def connect(user, code) do
+    github_post(code)
+    |> Tentacat.Client.new
+    |> Tentacat.Users.me
+    |> update_user(user)
+  end
+
+  defp update_user(%{github_id: _github_id} = github_info, user) do
+    associate(user, github_info)
```

Comment:
You mean, pass into `associate/2`? That would work for now, if that's the only thing we need to pass in. Later on, we might not have just the client ID but instead store a larger set of values, but we can worry about that when we come to it.

That being said, if associate already exists, and tests for it exist, no point i changing it now.